PKGBUILD

```diff
@@ -1,14 +1,14 @@
 # Maintainer: Chrysostomus @forum.manjaro.org
 pkgname=hibernator
-pkgver=0.1
+pkgver=0.2
```

Comment:
Nope, this is the version of the package. To update the pkgbuild maintaining the same package version, you are supposed to update `pkgrel` instead: https://wiki.archlinux.org/title/Arch_package_guidelines#Package_versioning

Reply:
I don't think anyone is interested in keeping the project alive anyway...